circle.yml

```diff
@@ -0,0 +1,53 @@
+machine:
+  python:
+    version: 2.7.3
+
+general:
+  artifacts:
+    - "reports"
+    - "test_root/log"
+
+dependencies:
+  override:
+    - npm install
+    - bundle install
+
+    - pip install setuptools==0.6c11
+    - pip install distribute==0.6.49
+    - pip install -r requirements/edx/paver.txt
+
+    # Mirror what paver install_prereqs does.
+    # After a successful build, CircleCI will
+    # cache the virtualenv at that state, so that
+    # the next build will not need to install them
+    # from scratch again.
+    - pip install -r requirements/edx/pre.txt
+    - pip install -r requirements/edx/github.txt
+    - pip install -r requirements/edx/local.txt
+
+    # HACK: within base.txt stevedore had a
+    # dependency on a version range of pbr.
+    # Install a version which falls within that range.
+    - pip install pbr==0.9.0
+    - pip install -r requirements/edx/base.txt
+    - pip install -r requirements/edx/post.txt
+
+    - pip install coveralls
+
+test:
+  override:
+    - ./scripts/all-tests.sh:
+        parallel: true
+
+  post:
+    - mkdir -p $CIRCLE_TEST_REPORTS/junit
+    # Copy the junit results up to be consumed by circleci,
+    # but only do this if there actually are results.
+    # Note that the greater than zero comparison is doing a
+    # string compare, but that should be fine for our purposes here.
+    - "if [ $(find reports -type f | wc -l) -gt 0 ] ; then cp -r reports/. $CIRCLE_TEST_REPORTS/junit ; fi":
+        parallel: true
+
+    # Configure your COVERALLS_REPO_TOKEN as an
```

Comment:
What's the impact of this not-being set? Should it be the first thing that's executed in this file?